odoo/sql_db.py

```diff
@@ -278,49 +237,48 @@ class Cursor(BaseCursor):
     """
     IN_MAX = 1000   # decent limit on size of IN queries - guideline = Oracle limit
 
-    def __init__(self, pool, dbname, dsn, serialized=True):
+    def __init__(self, pool, dbname, dsn, **kwargs):
         super().__init__()
+        if 'serialized' in kwargs:
+            warnings.warn("Since 16.0, 'serialized' parameter is depreciated")
```

Comment:
```suggestion
            warnings.warn("Since 16.0, 'serialized' parameter is deprecated", DeprecationWarning, 2)
```
the default warning is `UserWarning`.

Reply:
Also might be a good idea to have an `assert not kwargs`  afterwards to ensure no funny business, tho it seems unlikely (as normally people don't init cursors by hand) (in fact maybe the class should be left alone and cn.cursor is sufficient?)